### PR TITLE
A few bugfixes

### PR DESCRIPTION
**Pull request by remiolsen**
Some changes I made to make linting work on my python 3.6 installation

### CODE REVIEW

File: nf_core/lint.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def pf(file_path):
                 files = [files]
             if any([os.path.isfile(pf(f)) for f in files]):
                 self.passed.append((1, "File found: {}".format(files)))
-                self.files.append(f)
+                self.files.append(files)
```
Comment (sven1103) [inline nf_core/lint.py]: I think it should be `extend(files)`, as files will be a list.

File: nf_core/lint.py [hunk 2]
```diff
@@ -114,7 +114,7 @@ def pf(file_path):
...
```
Comment (sven1103) [inline nf_core/lint.py]: same here

File: nf_core/lint.py [hunk 3]
```diff
@@ -269,7 +269,7 @@ def check_readme(self):
             nf_config_version = self.config.get('params.nf_required_version').strip('\'"')
             try:
                 assert nf_badge_version == nf_config_version
-            except AssertionError, KeyError:
+            except (AssertionError, KeyError) as e:
```
Comment (sven1103) [inline nf_core/lint.py]: not necessary imo, cause the exception object is not accessed anywhere

**Comment by sven1103** [comment]
Thanks Remi! Could you also explain what you mean with "work on your Python 3.6" installation? Cause we are running CI with several Python versions. So if something is not working as expected, would be good to know (or even raise an issue) :)

File: nf_core/lint.py [hunk 4]
```diff
@@ -210,7 +210,8 @@ def check_config_vars(self):
             raise AssertionError("`nextflow config` returned non-zero error code: %s,\n   %s", e.returncode, e.output)
         else:
             for l in nfconfig_raw.splitlines():
-                k, v = str(l).split(' = ', 1)
+                ul = l.decode()
+                k, v = ul.split(' = ', 1)
```
Comment (sven1103) [inline nf_core/lint.py]: nice!

File: nf_core/lint.py (hunk 3 above)
Comment (remiolsen) [reply nf_core/lint.py]: You're right. I guess writing `as e` is more of a reflex for me 😄

**Comment by remiolsen** [comment]
Hi Sven. Thanks. I was testing just testing this tool on my Nextflow pipeline and using my the version of python I have install locally (`Python 3.6.3 :: Anaconda, Inc.`). It would throw two exceptions at various stages. `SyntaxError` on the exception block. And also another when making comparisons between byte-literals a unicode string.

**Comment by sven1103** [comment]
Would you mind opening an issue and describe this in detail (StackTrace etc.) Have to look into this, because the CI does not throw the exceptions :(

**Comment by remiolsen** [comment]
No worries, Sven. See #20

**Pull request merged**